gum/backend-linux/gumandroid.c

```diff
@@ -622,10 +623,12 @@ static const GumModuleDetails *
 gum_try_init_linker_details (void)
 {
   const GumModuleDetails * result = NULL;
+  GRegex * linker_path_pattern;
   gchar * linker_path, * maps, ** lines;
```

Comment:
```suggestion
  gchar * linker_path;
  GRegex * linker_path_pattern;
  gchar * maps, ** lines;
```

(So they're ordered by first-use, except for the variable holding the return value, which goes first.)